
message StripeInfo
{
    required int32 stripe_version = 1;
    optional uint32 level = 2;
    optional int32 field_type = 3;
    optional string root_name = 4;
    optional string root_type = 5;
}


